xml/depl_nodes.xml

```diff
@@ -7640,6 +7640,28 @@ radosgw_urls:
        No special configuration is needed for &o_comp;. It uses the
        configuration from the &o_blockstore; &barcl; to connect with volumes
        stored in SES.
+       For ephemeral storage, &o_comp; re-uses the <literal>rbd_store_user</literal>
+       and key from &o_blockstore; but has a separate <literal>rbd_store_pool</literal>
+       defined in the SES configuration.
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>RadosGW Integration</term>
+     <listitem>
+      <para>
+       Besides &o_blockstore;, SES cluster can also be used as a &o_objstore;
```

Comment:
&o_blockstore; will translate to Cinder, right? If so, I think it's better to say "block storage" which is a bit more abstract and matches "object storage" at the end of this sentence.

Reply:
https://github.com/SUSE-Cloud/doc-cloud/blob/maintenance/cloud_9/xml/entity-decl.ent#L270

Does indeed. At the end it translates to swift: https://github.com/SUSE-Cloud/doc-cloud/blob/maintenance/cloud_9/xml/entity-decl.ent#L95 

So it currently does match, but if you'd prefer the change I can do that.

Reply:
I meant the literal "object storage" we have at the end of the sentence on following line.

Reply:
I see. I can change that.